### ISSUE TITLE
Medicaments become more expensive after Napr quests

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#80]
After capturing of the concentration camp, most Flea Market goods get a discount, however, first aid kits and bandages become more expensive.

**Comment by Decane** [comment]
True for the 24-hour trader only - in configs/misc/trade/trade_trader_gar_digger_1.ltx:
```
[trader_generic_sell]

bandage				= 1.1, 1.2
medkit				= 1.1, 1.2
medkit_scientic			= 1.1, 1.2
medkit_army			= 1.1, 1.2
antirad				= 1.1, 1.2

[sell_after_liberate]:trader_generic_sell

bandage				= 1, 1.5
medkit				= 0.9, 1.5
medkit_scientic			= 1, 1.5
medkit_army			= 1, 1.5
antirad				= 1, 1.5
```

**Comment by Decane** [comment]
Fixed in ef4cd1e.
